### PR TITLE
Abd intention8

### PR DESCRIPTION
**Pull request by Abdeboskey**
#### What’s this PR do?
* Complete activity button functionality.
* Build and begin formatting Timer Display view.
* User now instantiates new Activity object using data from form, assigns it to "currentActivity" global variable and switches to Timer Display when user clicks "start activity" button.
* Refactored main.js to follow DRY and SRP conventions.

#### Where should the reviewer start?
* At "Start here" comment in main.js (line 10)

#### How should this be manually tested?
* Open index.html in browser, test buttons and input fields to make sure they work as intended.
* Console.log "currentActivity" after clicking "Start Activity" to make sure that new Activity object contains data from Input form.

#### Any background context you want to provide?
* These were some of the major hurdle features of this iteration!

#### Questions: (edited) 
* Will you please look at the refactored functions on lines 47-81,  and the startActivity and toggleElement functions on lines 12-22 of main.js and make sure that those make sense to you? If not, please ask for clarification in this PR and we can talk it out.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -11,61 +53,35 @@ function selectActivity(event) {
...
-}
-
-function meditateDefault(button, image) {
-  button.classList.add("btn-default");
-  button.classList.remove("meditate-selected");
-  image.src = "assets/meditate.svg";
-}
-
-function exerciseDefault(button, image) {
+function buttonDefault(button, image, category) {
```
Comment (NakiNorton) [inline main.js:77]: You did an excellent job refactoring six functions into one. This is a significant improvement!

**Review by NakiNorton — APPROVED** [review]
Really impressed with these changes.  It'll be great if we can find more ways to refactor like this. The 'Current Activity' page is starting to shape too. We should be able to finish this page in our session tomorrow.

File: main.js [hunk 2]
```diff
@@ -1,6 +1,48 @@
...
+  makeNewActivity(event);
+  toggleElement("new-activity-title", "new-activity-form");
+  toggleElement("current-activity-title", "timer-display");
+  // if statement to assign color to start button circle
+}
+
+function toggleElement(className1, className2) {
+  document.querySelector(`.${className1}`).classList.toggle(`hidden`);
+  document.querySelector(`.${className2}`).classList.toggle(`hidden`);
+}
```
Comment (NakiNorton) [inline main.js:22]: @Abdeboskey should `hidden` be in quotes instead of backticks since it's a class?

Comment (Abdeboskey) [reply main.js:22]: Great catch @NakiNorton. Functionally, I think they are doing the same thing, but for the sake of clarity and consistency we should absolutely use " " instead.

**Comment by jordy1611** [comment]
This looks pretty clean. Think our refactoring at the end of this iteration should include combing all the event listeners so that we can eliminate the rest of our global variables.

**Pull request merged**